tools/src/main/scala/scala/scalanative/codegen/llvm/CodeGen.scala

```diff
@@ -2,23 +2,18 @@ package scala.scalanative
 package codegen
 package llvm
 
-import java.io.File
-import java.nio.file.{Path, Paths, Files}
+import java.nio.file.{Files, Path, Paths}
 import scala.collection.mutable
-import scala.scalanative.build.Config
 import scala.scalanative.build.ScalaNative.{dumpDefns, encodedMainClass}
 import scala.scalanative.build.Build
 import scala.scalanative.io.VirtualDirectory
 import scala.scalanative.build
 import scala.scalanative.build.IO
 import scala.scalanative.linker.ReachabilityAnalysis
 import scala.scalanative.util.{Scope, partitionBy, procs}
-import java.nio.file.StandardCopyOption
-
-import scala.scalanative.codegen.{Metadata => CodeGenMetadata}
-import scala.concurrent._
+import scala.scalanative.codegen.Metadata as CodeGenMetadata
```

Comment:
Nice :-) and the next line too.

Reply:
why is scalafmt configured with scala3 dialect here? most CI builds are 2.12 or 2.13, and this accidental rewrite ended up breaking my tests :)

Reply:
Gosh, I guess when it changed we switched to scala3 and use scala213 for Scala 2 code only. What did it break? - I haven't had that problem.

Reply:
you haven't had this problem because you didn't try to rewrite to scala3 syntax. and intellij tries it :) also, scala2 code now assumes significant indentation and refuses to indent stuff correctly.